src/inverseRobot.js

```diff
@@ -24,7 +24,26 @@
  * @return {object}
  */
 function inverseRobot(robot) {
-  // write code here
+  const newObj = {};
+  const arr = [];
+
+  for (const key in robot) {
+    arr.push(robot[key]);
+  };
+
+  const newArr = arr.sort();
+
+  for (let i = 0; i < newArr.length - 1; i++) {
+    if (newArr[i] === newArr[i + 1]) {
+      return null;
+    };
+  };
```

Comment:
you can use `hasOwnProperty` instead of code from 30 to 40 line
<img width="365" alt="image" src="https://user-images.githubusercontent.com/99652389/183838756-6f504890-e8e4-4d67-9caa-8363d06c2b8d.png">